Debug templates

AriettaZ opened the pull request:
## Not Working:
- Basic_ABAQUS_Job_Owens :
Abaqus Error: License for explicit is not available

- Basic_ANSYS_Mechanical_Owens & Basic_ANSYS_FLUENT_Parallel_Job_Owens &  Basic_ANSYS_FLUENT_Serial_Job_Owens &Basic_ANSYS_CFX_Job_Owens
User/host not on INCLUDE list for feature.
Feature:       aa_t_a
License path:  1055@license5.osc.edu:
FlexNet Licensing error:-39,147

- Simple_Parallel_Job & Simple_OMP_MPI_Job
`[proxy:0:0@o0559.ten.osc.edu] HYDU_create_process (utils/launch/launch.c:75): execvp error on file a.out (No such file or directory)` 
a.out is copied from  $PBS_O_WORKDIR to $TMPDIR and I tried to use different files but still have this error. Not sure how to solve. 

## Working:
- Basic_BLAST_Job_Owens
- Simple_OMP_Job
- Basic_AMBER_Job_Owens
- Basic COMSOL Parallel Job Owens & Basic_COMSOL_Serial_Job_Owens
COMSOL file from: http://csa.phys.uvic.ca/teaching/multi-physics-projects-with-comsol/electromagnetic-theory/solution-of-electrostatics-and-magnetostatics-problems-with-comsol/comsol-mph-files-with-the-solution-of-the-problems/laplace-s-equation/view

**I also fixed outdated urls mentioned in #6, removed Simple_Abaqus_Job mentioned in #7 and updated naming of ANSYS, ANSYS_CFX and ANSYS_FLUENT filenames mentioned in #8.**

ericfranz commented on ondemand.osc.edu/apps/myjobs/templates/Basic_AMBER_Job_Owens/basic_amber.sh:
```diff
@@ -1,22 +1,61 @@
+#PBS -N jac9999cuda.owens
+#PBS -j oe
+#PBS -m ae
+#PBS -M srb@osc.edu
```
I think that we don't want to have Scott get emails every time people submit jobs :-)

ericfranz commented on ondemand.osc.edu/apps/myjobs/templates/Basic_AMBER_Job_Owens/basic_amber.sh:
```diff
@@ -1,22 +1,61 @@
-#PBS -l walltime=0:20:00
+# AMBER 16 Example Batch Script for Owens
+#
+#PBS -N jac9999cuda.owens
```
Maybe the job name can be something more appropriate? "ondemand/myjobs/basic_amber" for example?

ericfranz replied:
actually i wonder what we do for interactive apps => might be nice to have the same job name convention

ericfranz commented:
Several things:

1. Ask Summer about these jobs: Simple_Parallel_Job & Simple_OMP_MPI_Job?
2. The convention used for job names in interactive apps is "ondemand/sys/dashboard/sys/bc_osc_jupyter" where the app is "ondemand/sys/dashboard" and the batch connect app is "sys/bc_osc_jupyter"; in this case the app is "ondemand/sys/myjobs" and then w have individual scripts such as: "basic_amber" so I think a reasonable default job would be "ondemand/sys/myjobs/basic_amber"
3. I requested you to get access to Abaqus and Ansys. Hopefully that will go through today.

ericfranz commented:
@laser505t did you have any template modifications or did Ariel take care of all of them?

ericfranz commented:
Finally, once you have completed everything and the review looks good you will need to make a recursive copy of these templates to the other directories that require the copy. You can run `rake` in the root to run the tests necessary, or just see the Travis CI build failures :-).

AriettaZ commented:
Updates:
## Not Working:
- Basic_ANSYS_Mechanical_Owens & Basic_ANSYS_FLUENT_Parallel_Job_Owens &  Basic_ANSYS_FLUENT_Serial_Job_Owens &Basic_ANSYS_CFX_Job_Owens
  Waiting for input files from Summer

## Working:
- Simple_Parallel_Job & Simple_OMP_MPI_Job
- Basic_ABAQUS_Job_Owens
- Basic_BLAST_Job_Owens
- Simple_OMP_Job
- Basic_AMBER_Job_Owens
- Basic COMSOL Parallel Job Owens & Basic_COMSOL_Serial_Job_Owens
COMSOL file from: http://csa.phys.uvic.ca/teaching/multi-physics-projects-with-comsol/electromagnetic-theory/solution-of-electrostatics-and-magnetostatics-problems-with-comsol/comsol-mph-files-with-the-solution-of-the-problems/laplace-s-equation/view

**I also fixed outdated urls mentioned in #6, removed Simple_Abaqus_Job mentioned in #7 and updated naming of ANSYS, ANSYS_CFX and ANSYS_FLUENT filenames mentioned in #8.**

AriettaZ commented:
Tyler and I split the templates like this:

### Ariel: (debug-template branch)
Simple_OMP_Job
Simple_OMP_MPI_Job
Simple_Parallel_Job
Basic_ABAQUS_Job_Owens
Basic_AMBER_Job_Oakley
Basic_AMBER_Job_Owens
Basic_ANSYS_Mechanical_Owens 
Basic_ANSYS_FLUENT_Parallel_Job_Owens 
Basic_ANSYS_FLUENT_Serial_Job_Owens 
Basic_ANSYS_CFX_Job_Owens
Basic_BLAST_Job_Oakley
Basic_BLAST_Job_Owens
Basic_COMSOL_Parallel_Job_Owens
Basic_COMSOL_Serial_Job_Owens
**_This pull request fixed the templates above._**

### Tyler: (template-fix branch)
QChem_job
Simple_MPI_Job
Simple_Seqential_Job
Basic_Gaussian_Oakley
Basic_Gaussian_Owens
Basic_MATLAB_Oakley
Basic_MATLAB_Owens
Basic_Octave_Oakley
Basic_Octave_Owens
Basic_OpenFOAM_Parallel_Job_Oakley
Basic_OpenFOAM_Serial_Job_Oakley
Basic_Star-CCM_Parallel_Job_Owens
Basic_Star-CCM_Serial_Job_Owens
Simple_Abaqus_Job
default
 
